components/workflow.rst

```diff
@@ -84,7 +84,7 @@ you can retrieve a workflow from it and use it as follows::
     // ...
     // Consider that $blogPost is in place "draft" by default
     $blogPost = new BlogPost();
-    $workflow = $registry->get($blogPost);
+    $workflow = $registry->get(BlogPost::class);
```

Comment:
you can not change this like that as a previous version of sf will break
you must add a versionadded directive instead documentation this new way available :)